.github/workflows/backend-dev-cd.yml

```
@@ -34,76 +34,47 @@
-              echo "서버가 정상적으로 실행되지 않아 종료함"
-              exit 1
-            fi
-          done
-
-          echo "-- 배포 종료 --"
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: .env 파일 생성
+        working-directory: /home/ssm-user/coursepick
+        run: | 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)